lib/CMakeLists.txt

```diff
@@ -3,11 +3,11 @@ add_library(cil ImageInfo.cpp ImageMatrix.cpp Pixel.cpp Transformations.cpp)
 target_include_directories(cil PUBLIC ${CMAKE_SOURCE_DIR}/include)
 
 if (CIL_ENABLE_JPEG)
-    target_compile_definitions(cil PRIVATE CIL_JPEG_ENABLED)
+    target_compile_definitions(cil PUBLIC CIL_JPEG_ENABLED)
```

Comment:
Is this an intended change? I remember you had converted this as PRIVATE in the one of the previous PRs

Reply:
Yes, it is an intended change. It is done so that (test) executables that link to `cil` target automatically inherit these compile definitions.